dev/Materials/Acrylic/AcrylicBrush_19h1_themeresources.xaml

```diff
@@ -35,6 +35,16 @@
                 FallbackColor="#202020"
                 BackgroundSource="HostBackdrop"/>
             <SolidColorBrush x:Key="SystemControlDefaultBrighteningBrush" Color="#FFFFFF" Opacity="0.0419"/>
```

Comment:
You can remove this brush.

LayerFillColorDefaultBrush and AccentLayerFillColorDefaultBrush will be used instead for layers over acrylic.

@YuliKl for awareness.

Reply:
Okay, removed the brush now.